ldn-plugin.js

```diff
@@ -91,7 +91,9 @@ var Ldn = {
     $(document).on("click", ".supplement", function(e){
       var url = $(e.target).attr("data-url");
       var id = $(e.target).attr("data-id");
-      _this.retrieveData(url, id);
+      _this.retrieveData(url, id).done(function(){
+        $(e.target).parents("li").remove();
```

Comment:
What does this do?

Reply:
When the `object` is successfully retrieved, it is removed from the bell list. That prevents clutter or double-applications of lists. If preferred, it could be disabled instead so you can still see the logo and rights information.

Reply:
awesome. much needed.